Index: src/main/java/iktlab/javaPro/j4Group/Projects/One/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package iktlab.javaPro.j4Group.Projects.One;\r\n\r\npublic class Main {\r\n\r\n    public static void main(String[] args) {\r\n        Pet dog = new Pet();\r\n        dog.species = \"dog\";\r\n        dog.nickname = \"Rock\";\r\n        dog.age = 5;\r\n        dog.tricklevel = 75;\r\n        dog.habits = new String[]{\"eat\", \"drink\", \"sleep\"};\r\n\r\n\r\n    }\r\n}\r\n
===================================================================
diff --git a/src/main/java/iktlab/javaPro/j4Group/Projects/One/Main.java b/src/main/java/iktlab/javaPro/j4Group/Projects/One/Main.java
--- a/src/main/java/iktlab/javaPro/j4Group/Projects/One/Main.java	
+++ b/src/main/java/iktlab/javaPro/j4Group/Projects/One/Main.java	
@@ -3,12 +3,14 @@
 public class Main {
 
     public static void main(String[] args) {
-        Pet dog = new Pet();
-        dog.species = "dog";
-        dog.nickname = "Rock";
-        dog.age = 5;
-        dog.tricklevel = 75;
-        dog.habits = new String[]{"eat", "drink", "sleep"};
+        Pet Rock = new Pet("dog", "Rock" , (byte) 5,(byte) 75,
+                new String[]{"eat","drink", "sleep"});
+        System.out.println(Rock);
+        Human Vito = new Human("Vito", "Karleone", (short) 1957);
+        Human Jane = new Human("Jane", "Karleone", (short) 1958);
+        Human Michael = new Human("Michael", "Karleone",
+                (short) 1977, (byte) 90, Rock,Jane,Vito, null);
+        System.out.println(Michael);
 
 
     }
Index: src/main/java/iktlab/javaPro/j4Group/Projects/One/Human.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package iktlab.javaPro.j4Group.Projects.One;\r\n\r\npublic class Human {\r\n    public String name;\r\n    public String surname;\r\n    public int year;\r\n    public int iq;\r\n    static Pet pet = new Pet();\r\n    public Human mother = new Human();\r\n\r\n    public Human father = new Human();\r\n    public String[][] schedule = new String[7][];\r\n    static void greetPet(){\r\n        System.out.println(\"Hello, \"+ pet.nickname);\r\n    }\r\n    static void describePet(){\r\n        System.out.print(\"I have a \"+pet.species);\r\n        System.out.print(\", he is \"+pet.age);\r\n        if (pet.tricklevel>50){\r\n            System.out.print(\", he is very sly \");\r\n        }\r\n        else {\r\n            System.out.print(\", he is almost not sly \");\r\n        }\r\n\r\n    }\r\n\r\n\r\n    }\r\n\r\n\r\n
===================================================================
diff --git a/src/main/java/iktlab/javaPro/j4Group/Projects/One/Human.java b/src/main/java/iktlab/javaPro/j4Group/Projects/One/Human.java
--- a/src/main/java/iktlab/javaPro/j4Group/Projects/One/Human.java	
+++ b/src/main/java/iktlab/javaPro/j4Group/Projects/One/Human.java	
@@ -1,31 +1,58 @@
 package iktlab.javaPro.j4Group.Projects.One;
 
+import java.util.Arrays;
+
 public class Human {
     public String name;
     public String surname;
-    public int year;
-    public int iq;
+    public short dateOfbirth;
+    public byte iq;
     static Pet pet = new Pet();
-    public Human mother = new Human();
+    public Human mother;
+    public Human father;
+    public String[][] schedule ;
+
+    public void greetPet(){
+        System.out.printf("Hello,  %s"+ pet.nickname);
+    }
+    public void describePet(){
+        String sly = pet.tricklevel>=50 ? "very sly" : "almost not sly";
+        System.out.printf("I have a %s, he is %s years old, he is %s",pet.species,pet.age,sly);
+
+    }
 
-    public Human father = new Human();
-    public String[][] schedule = new String[7][];
-    static void greetPet(){
-        System.out.println("Hello, "+ pet.nickname);
+    @Override
+    public String toString() {
+        return "Human{" +
+                "name='" + name + '\'' +
+                ", surname='" + surname + '\'' +
+                ", dateOfbirth=" + dateOfbirth +
+                ", iq=" + iq +
+                ", mother=" + mother +
+                ", father=" + father +
+                ", schedule=" + Arrays.toString(schedule) +
+                '}';
     }
-    static void describePet(){
-        System.out.print("I have a "+pet.species);
-        System.out.print(", he is "+pet.age);
-        if (pet.tricklevel>50){
-            System.out.print(", he is very sly ");
-        }
-        else {
-            System.out.print(", he is almost not sly ");
-        }
+    public Human(){
+
+    }
+    public Human(String name, String surname, short dateOfbirth){
+
+    }
+    public Human(String name, String surname,short dateOfbirth, byte iq, Pet pet,
+                 Human mother, Human father, String[][] schedule){
+        this.name = name;
+        this.surname = surname;
+        this.dateOfbirth = dateOfbirth;
+        this.iq = iq;
+        this.pet = pet;
+        this.mother = mother;
+        this.father = father;
+        this.schedule = schedule;
 
     }
 
 
-    }
+}
 
 
Index: src/main/java/iktlab/javaPro/j4Group/Projects/One/Pet.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package iktlab.javaPro.j4Group.Projects.One;\r\n\r\nimport static iktlab.javaPro.j4Group.Projects.One.Human.pet;\r\n\r\npublic class Pet {\r\n    public String species;\r\n    public String  nickname;\r\n    public int age;\r\n    public int tricklevel;\r\n    public String[] habits = new String[5];\r\n\r\n    static void eat(){\r\n        System.out.println(\"I am Eating\");\r\n    }\r\n\r\n    static void  respond(){\r\n\r\n        System.out.println(\"Hello, owner. I am \"+ pet.nickname+\". I miss you\");\r\n    }\r\n    static void foul(){\r\n        System.out.println(\"I need to cover it up\");\r\n    }\r\n\r\n\r\n\r\n\r\n}\r\n
===================================================================
diff --git a/src/main/java/iktlab/javaPro/j4Group/Projects/One/Pet.java b/src/main/java/iktlab/javaPro/j4Group/Projects/One/Pet.java
--- a/src/main/java/iktlab/javaPro/j4Group/Projects/One/Pet.java	
+++ b/src/main/java/iktlab/javaPro/j4Group/Projects/One/Pet.java	
@@ -1,27 +1,53 @@
 package iktlab.javaPro.j4Group.Projects.One;
 
+import java.util.Arrays;
+
 import static iktlab.javaPro.j4Group.Projects.One.Human.pet;
 
 public class Pet {
     public String species;
     public String  nickname;
-    public int age;
+    public byte age;
     public int tricklevel;
-    public String[] habits = new String[5];
+    public String[] habits ;
 
-    static void eat(){
+    public void eat(){
         System.out.println("I am Eating");
     }
 
-    static void  respond(){
+    public void  respond(){
 
-        System.out.println("Hello, owner. I am "+ pet.nickname+". I miss you");
+        System.out.printf("Hello, owner. I am %s. I miss you",pet.nickname);
     }
-    static void foul(){
+    public void foul(){
         System.out.println("I need to cover it up");
     }
 
+    @Override
+    public String toString() {
+        return "Pet{" +
+                "species='" + species + '\'' +
+                ", nickname='" + nickname + '\'' +
+                ", age=" + age +
+                ", tricklevel=" + tricklevel +
+                ", habits=" + Arrays.toString(habits) +
+                '}';
+    }
+    public Pet(){
 
+    }
+    public  Pet(String species, String nickname){
+        this.species = species;
+        this.nickname = nickname;
 
+    }
+    public Pet(String species, String nickname, byte age,
+               byte tricklevel, String[] habits){
+        this.species = species;
+        this.nickname = nickname;
+        this.age = age;
+        this.tricklevel = tricklevel;
+        this.habits = habits;
 
+    }
 }
